### DISCUSSION TITLE
On average, sloths move how many meters per day?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#366] [Galaxy Trivia] [answered]
**Hint:** They don't get their name for nothing ...

**Comment by Roberto-Larivera** [comment]
Di solito ì Bradipi percorrono oltre 40 metri al giorno

**Reply by ParanoidUser** [reply]
Bravo! 👏

**Comment by peterbaccaro** [comment]
Sloths generally travel no more than 38 meters in a single day.

**Comment by thuongtruong1009** [comment, marked as answer]
Sloths are known for their slow movements and low activity levels. On average, a sloth will only move a few meters per day, typically around 2 to 4 meters, although this can vary depending on factors such as the species of sloth, their age, health, and the availability of food and water. Sloths are adapted to conserve energy and move slowly through the trees, spending most of their time sleeping and resting. Their slow movement also helps them avoid predators and conserve energy, making them highly efficient at surviving in their habitat.
